src/sentry/static/sentry/app/views/projectInstall/createProject.jsx

```diff
@@ -66,47 +67,40 @@ class CreateProject extends React.Component {
   }
 
   componentDidMount() {
-    const {organization} = this.props;
-    const alertDefaultsExperimentVariant =
-      organization.experiments?.AlertDefaultsExperiment;
-    const isInAlertDefaultsExperiment = [
-      '2OptionsV1',
-      '3OptionsV2',
-      'controlV1',
-    ].includes(alertDefaultsExperimentVariant);
-
-    if (isInAlertDefaultsExperiment) {
-      logExperiment({
-        organization,
-        key: 'AlertDefaultsExperiment',
-      });
-    }
+    const {organization, experimentAssignment} = this.props;
 
-    const analyticsEventOptions = {
+    trackAnalyticsEvent({
       eventKey: 'new_project.visited',
       eventName: 'New Project Page Visited',
-      organization_id: parseInt(this.props.organization.id, 10),
-    };
-    if (isInAlertDefaultsExperiment) {
-      analyticsEventOptions.alert_defaults_experiment_variant = alertDefaultsExperimentVariant;
-    }
-    trackAnalyticsEvent(analyticsEventOptions);
+      organization_id: organization.id,
+      alert_defaults_experiment_variant: experimentAssignment,
+    });
+  }
+
+  get issueAlertOptionsEnabled() {
+    return ['2OptionsV1', '3OptionsV2'].includes(this.props.experimentAssignment);
   }
 
-  renderProjectForm = (
-    projectName,
-    team,
-    teams,
-    platform,
-    hasIssueAlertOptionsEnabled,
-    organization,
-    canSubmitForm
```

Comment:
I prefer pulling things from state / props where they're needed rather than passing parameters around.  This made it pretty hard to follow what was going on here.... so I just removed all of them.